### ISSUE TITLE
Download button doesn't work in MS Edge 38

### ISSUE THREAD

**Issue by daveloyall** [robinmoisson/staticrypt#5]
When I click the Download button in Edge, I get a new blank tab with no content in the address bar.  That's it.

Let me know if I can help by following troubleshooting instructions you provide!

**Comment by bfred-it** [comment]
Is the file downloaded though?

Perhaps only this is necessary:

```js
if (typeof document.createElement('a').download === 'undefined') {
  document.querySelector('a.download').target = '_blank'
}
```

And removing `target="_blank"` from index.html

**Comment by robinmoisson** [comment]
Thanks @daveloyall, this should be fixed at the same time as #4 (+ `ŧarget` attribute is removed for everyone).

Do you let me know if that works for you !

**Comment by daveloyall** [comment]
Thanks you two!  It works now.
